lib/auth/grpcserver.go

```diff
@@ -1135,6 +1135,73 @@ func (g *GRPCServer) DeleteAllDatabaseServers(ctx context.Context, req *proto.De
 	return &emptypb.Empty{}, nil
 }
 
+// GetAllDatabaseServices returns all registered DatabaseServices.
+func (g *GRPCServer) GetAllDatabaseServices(ctx context.Context, _ *proto.GetAllDatabaseServicesRequest) (*proto.DatabaseServiceV1List, error) {
```

Comment:
`GetAllDatabaseServices -> GetDatabaseServices` 

The `All` word seems to be superfluous and  introduces a new naming style inconsistent with older endpoints.

Reply:
This might be removed as suggested here:
https://github.com/gravitational/teleport/pull/19469#discussion_r1052248625

I'll make sure to not backport this PR before we resolve this comment

Reply:
`GetAllDatabaseServices` removed
https://github.com/gravitational/teleport/pull/19469#discussion_r1052444877